lang/Lib/gflanguages/data/scripts/Hani.textproto

```diff
@@ -1,3 +1,6 @@
+# Hani is a set of characters, which includes not only Simplified and Traditional Chinese,
+# but also Japanese Kanji and Korean Hanja.
+# So to avoid unexpected characters users might see from sample texts,
```

Comment:
nit: please remove the second sentence. that is an implementation detail for our usage of this dataset.